### PR TITLE
Checkstyle

### PR DESCRIPTION
**Pull request by EnKarin**
Fixed warnings CustomImportOrder, WhitespaceAfter, WhitespaceAround, Indentation, EmptyLineSeparator

### CODE REVIEW

File: src/main/java/io/github/enkarin/bookcrossing/admin/service/AdminService.java [hunk 1]
```diff
@@ -19,7 +19,9 @@
 public class AdminService {
 
     private final UserRepository userRepository;
+
```
Comment (mfvanek) [inline src/main/java/io/github/enkarin/bookcrossing/admin/service/AdminService.java:22]: These empty lines look redundant. I'd recommend to exclude corresponding checkstyle rule

File: src/main/java/io/github/enkarin/bookcrossing/books/controllers/MyBookController.java [hunk 2]
```diff
@@ -23,6 +23,7 @@
 import java.util.Objects;
 import java.util.Optional;
 
+
```
Comment (mfvanek) [inline src/main/java/io/github/enkarin/bookcrossing/books/controllers/MyBookController.java:26]: This empty line looks redundant. I'd recommend to exclude corresponding checkstyle rule

**Review by mfvanek — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 3]
```diff
@@ -140,6 +140,14 @@
                 <configuration>
```
Comment (mfvanek) [inline pom.xml:140]: [WARNING] Old version of checkstyle detected. Consider updating to >= v8.30
[WARNING] For more information see: https://maven.apache.org/plugins/maven-checkstyle-plugin/examples/upgrading-checkstyle.html

File: pom.xml [hunk 4]
```diff
@@ -140,6 +140,14 @@
...
                     <configLocation>/config/checkstyle.xml</configLocation>
                 </configuration>
+                <executions>
+                    <execution>
+                        <id>checkstyle</id>
+                        <goals>
+                            <goal>check</goal>
```
Comment (mfvanek) [inline pom.xml:147]: Default phase is "verify", and it's already too late. Checkstyle should be run with compilation

**Review by mfvanek — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (EnKarin) [reply pom.xml]: Fixed

File: pom.xml (hunk 4 above)
Comment (EnKarin) [reply pom.xml:145]: Verification added

**Review by mfvanek — APPROVED** [review]

**Pull request merged**